Add GDCH provider and plan (#3318)
kyma-project/kyma-environment-broker: docs/user/03-10-service-description.md

| **additionalWorkerNodePools<sup>1</sup>** | array  | Defines a custom list of additional worker node pools.                                     |    No    | None          |

</details>

You can configure the following provisioning parameters for Google Distributed Cloud Hosted (GDCH):

<details>
<summary label="gdch-plan">
Google Distributed Cloud Hosted
</summary>

> > **Note:** The following parameters are not available for the `gdch` plan and are disabled by design: custom **networking** configuration, **colocate control plane**, and **GVisor** container runtime.

Reviewer: ```suggestion
> ### Note:
> The custom **networking** configuration parameter is not available for the `gdch` plan and is disabled by design.
```

Reviewer: The other two params - **colocate control plane** and **GVisor**(internal) - are not mentioned in this doc.

| **region**                    | string | Defines the cluster region.                                                                |   Yes    | None                   |
| **zones**                     | string | Defines the list of zones in which KIM creates a cluster.                                  |    No    | `["us-west16-a"]`      |
| **autoScalerMin<sup>1</sup>** | int    | Specifies the minimum number of virtual machines to create.                                |    No    | `3`                    |
| **autoScalerMax<sup>1</sup>** | int    | Specifies the maximum number of virtual machines to create.                                |    No    | `10`                   |
| **maxSurge<sup>1</sup>**      | int    | Specifies the maximum number of virtual machines that are created during an update.        |    No    | `4`                    |
| **maxUnavailable<sup>1</sup>**            | int    | Specifies the maximum number of virtual machines that can be unavailable during an update. |    No    | `1`                    |

> | **additionalWorkerNodePools<sup>1</sup>** | array  | Defines a custom list of additional worker node pools.                                     |    No    | None                   |

Author: This looks like internal default not exposed by the schema.
